lib/bungee/src/Partials.cpp

```
@@ -0,0 +1,62 @@
+// Copyright (C) 2020-2026 Parabola Research Limited
+// SPDX-License-Identifier: MPL-2.0
+
+#include "Partials.h"
+
+namespace Bungee::Partials {
+
+void enumerate(std::vector<Partial> &partials, int n, Eigen::Ref<Eigen::ArrayX<float>> energy)
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)